src/modules/mhct-lookup.js

```diff
@@ -0,0 +1,391 @@
+    return await fetch(url)
+        .then(response => response.json())
+        .catch(err => {
+            Logger.log(`findThings: Error getting item ${qsOptions.toString()} - ${err}`);
+        });
+}
+
+/**
+ * Initialize (or refresh) a list of items from MHCT
+ * @param {'mouse'|'loot'} type The type of thing to get a list of
+ * @param {Array} list The list to populate / re-populate
+ */
+async function getMHCTList(type, list) {
+    const now = DateTime.utc();
+    if (type && refresh_list[type]) {
+        const next_refresh = refresh_list[type].plus(refresh_rate);
+        if (now < next_refresh)
+            return [];
+        refresh_list[type] = now;
+    } else {
+        Logger.log(`getMHCTList: Received a request for ${type} but I don't do that yet`);
+    }
+    Logger.log(`MHCT list: Getting a new ${type} list`);
+    const url = `https://mhhunthelper.agiletravels.com/searchByItem.php?item_type=${type}&item_id=all`;
+    await fetch(url)
+        .then(response => (response.status === 200) ? response.json() : '')
+        .then((body) => {
+            if (body) {
+                Logger.log(`MHCT: Got a new ${type} list`);
+                Array.prototype.push.apply(list, body);
+                list.forEach(item => item.lowerValue = item.value.toLowerCase());
+            }
+        });
+    Logger.log(`MHCT List: ${type} was ${list.length} long`);
+}
+
+/**
+ * Initialize (or refresh) the known filters from @devjacksmith's tools.
+ * @returns {Promise<void>}
+ */
+async function getFilterList() {
+    const now = DateTime.utc();
+    if (refresh_list.filter) {
+        const next_refresh = refresh_list.filter.plus(refresh_rate);
+        if (now < next_refresh)
+            return Promise.resolve();
+    }
+    refresh_list.filter = now;
+
+    Logger.log('Filters: Requesting a new filter list.');
+    const url = 'https://mhhunthelper.agiletravels.com/filters.php';
+    return fetch(url).then(response => (response.status === 200) ? response.json() : '').then((body) => {
+        if (body) {
+            Logger.log('Filters: Got a new filter list');
+            filters.length = 0;
+            Array.prototype.push.apply(filters, body);
+            filters.forEach(filter => filter.lowerValue = filter.code_name.toLowerCase());
+        } else {
+            Logger.warn('Filters: request returned non-200 response');
+        }
+    }).catch(err => Logger.error('Filters: request returned error:', err));
+}
+
+async function initialize() {
+    if (someone_initialized)
+        return true;
+    someone_initialized = true;
+    await getMHCTList('mouse', mice);
+    await getMHCTList('loot', loot);
+    await getFilterList();
+    Logger.log(`MHCT Initialized: Loot: ${loot.length}, mice: ${mice.length}, filters: ${filters.length}`);
+    return true;
+}
+
+module.exports.getMHCTList = getMHCTList;
+module.exports.initialize = initialize;
```

Comment:
Looks like I could also just `export <thing>`. I guess CJS has module.exports. ES6 has `export` keyword. I agree, not a huge difference but less typing in the future if they all dropped module.

Reply:
We use CommonJS modules, rather than ESM. So without switching the module type,  `import ____ from '....';` and `export (default) _____` won't compile. I don't recommend we switch until they're no longer experimental on the current LTS (which is at least node 14).